exporter/awsxrayexporter/internal/translator/segment.go

```diff
@@ -46,6 +47,14 @@ var (
 	reInvalidSpanCharacters = regexp.MustCompile(`[^ 0-9\p{L}N_.:/%&#=+,\-@]`)
 )
 
+var (
+	remoteXrayExporterDotConverter = featuregate.GlobalRegistry().MustRegister(
+		"xray.exporter.dotconverter.disabled",
```

Comment:
```suggestion
		"exporter.xray.allowDot",
```
I believe that having the `.disabled` in the feature gate name could be confusing to end users. End users will enable or disable this feature on their own in the command line. I have suggested a new name which I think will provide a bit more clarity and align with the default behavior.